plugins/map/EditingSupport.jsx

```diff
@@ -58,7 +58,7 @@ class EditingSupport extends React.Component {
     }
     render() {
         if (this.state.showRecordLocation) {
-            const geomType = this.props.editContext.geomType.replace(/Z$/, '');
+            const geomType = this.props.editContext.geomType?.replace(/Z$/, '') || '';
```

Comment:
What about
```
if (this.state.showRecordLocation && this.props.editContext.geomType) {
```
Using the `LocationRecorder` with an empty `geomType` will probably not work.